src/test/java/org/jabref/logic/exporter/CffExporterTest.java

```diff
@@ -62,25 +54,26 @@ public final void exportsCorrectContent(@TempDir Path tempFile) throws Exception
         cffExporter.export(databaseContext, file, Collections.singletonList(entry));
 
         List<String> expected = List.of(
-        "cff-version: 1.2.0",
-        "message: \"If you use this, please cite the work from preferred-citation.\"",
-        "authors:",
-        "  - name: Test Author",
-        "title: Test Title",
-        "preferred-citation:",
-        "  type: article",
-        "  authors:",
-        "    - name: Test Author",
-        "  title: Test Title",
-        "  url: \"http://example.com\"");
+                "cff-version: 1.2.0",
+                "message: If you use this software, please cite it using the metadata from this file.",
+                "title: Test Title",
+                "authors:",
+                "  - family-names: Author",
+                "    given-names: Test",
+                "preferred-citation:",
+                "  type: article",
+                "  authors:",
+                "    - family-names: Author",
+                "      given-names: Test",
+                "  title: Test Title",
+                "  url: http://example.com");
 
         assertEquals(expected, Files.readAllLines(file));
     }
 
     @Test
     public final void usesCorrectType(@TempDir Path tempFile) throws Exception {
         BibEntry entry = new BibEntry(StandardEntryType.InProceedings)
-                .withCitationKey("test")
```

Comment:
Please keep the key, because it is a mandatory field. The export should not fail if a key is present.